wgpu-types/src/lib.rs

```diff
@@ -944,8 +944,7 @@ impl DepthStencilStateDescriptor {
 /// Format of indices used with pipeline.
 #[repr(C)]
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
-#[cfg_attr(feature = "trace", derive(Serialize))]
-#[cfg_attr(feature = "replay", derive(Deserialize))]
+#[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
```

Comment:
Curious why this change was needed?

Reply:
There was one or two checks complaining about it. I don't quite remember which ones.